addons/mrp/tests/test_procurement.py

```diff
@@ -183,17 +183,20 @@ def test_date_propagation(self):
         self.assertEqual(mo_form.product_qty, 10, 'the quantity to produce is not good relative to the move')
 
         mo = mo_form.save()
-
         # Confirming mo create finished move
         move_orig = self.env['stock.move'].search([
             ('move_dest_ids', 'in', move_dest.ids)
         ], limit=1)
+        move_orig.write({'propagate_date': True, 'propagate_date_minimum_delta': 1})
 
         self.assertEqual(len(move_orig), 1, 'the move orig is not created')
         self.assertEqual(move_orig.product_qty, 10, 'the quantity to produce is not good relative to the move')
 
         move_dest_scheduled_date = move_dest.date_expected
         mo.date_planned_start += timedelta(days=5)
+
+        self.assertAlmostEqual(move_orig.date_expected, move_dest_scheduled_date + timedelta(days=4, hours=23), delta=timedelta(seconds=1))
```

Comment:
add a comment: check that writing on the MO planned date do change the move finished date_expected